syntax = "proto3";

// sin cos + - / *

message Arguments2
{
  double arg1 = 1;
  double arg2 = 2;
}

message Result
{
  double value = 1;
}

message FailResult
{
  string error_message = 1;
}